packages/core/src/js/feedback/FeedbackForm.styles.ts

```diff
@@ -79,7 +79,22 @@ const defaultStyles: FeedbackFormStyles = {
   },
   modalBackground: {
     flex: 1,
-    justifyContent: 'center',
+    justifyContent: 'flex-end',
+    backgroundColor: 'rgba(0, 0, 0, 0.8)',
+  },
+  modalSheetContainer: {
```

Comment:
Note: If we proceed with this approach the new style should be added [in the docs](https://github.com/getsentry/sentry-docs/pull/12503/files#diff-f152431fe6cca40aaef7918e3e1b44445e7b7c56506201274fa128c5a6bbf024R119)